packages/admin/cms-admin/src/dam/DataGrid/label/DamItemLabel.tsx

```diff
@@ -29,7 +31,7 @@ const Path = styled(Typography)`
 `;
 
 interface DamItemLabelProps {
-    asset: GQLDamFolderTableFragment | GQLDamFileTableFragment;
+    asset: DamRow;
```

Comment:
I don't think it's a good idea that all components in the grid use the fragment from the parent, but fixing that is out of scope...

Reply:
true